### PR TITLE
[GLUTEN-5773] Update aws-sdk-cpp version to 1.11.285 (from 1.11.169)

### PR DESCRIPTION
**Pull request by yma11**
## What changes were proposed in this pull request?

update aws-sdk-cpp to version 1.11.285

(Fixes: \#5773)

## How was this patch tested?
N/A

### CODE REVIEW

**Comment by yma11** [comment]
@PHILO-HE  please help take a review. Thanks.

**Comment by FelixYBW** [comment]
So we only updated the the aws-sdk-cpp in vcpkg.

@PHILO-HE Is there any risk if we only update individual components in vcpkg but not vcpkg itself in long term?

**Comment by PHILO-HE** [comment]
> So we only updated the the aws-sdk-cpp in vcpkg.
> 
> @PHILO-HE Is there any risk if we only update individual components in vcpkg but not vcpkg itself in long term?

@FelixYBW, I think there is no risk. The only inconvenience is, we have to copy the ports files for some lib from vcpkg and maintain them in Gluten. I can take a try to upgrade vcpkg.

**Comment by PHILO-HE** [comment]
@yma11, I note this lib is upgraded to 1.11.321 in upstream velox. Do we need to align with it in Gluten?
https://github.com/facebookincubator/velox/pull/9756

**Review by PHILO-HE — DISMISSED** [review]
Looks good!

**Comment by yma11** [comment]
> Looks good!

Thanks but wait for a while. May need some extra change. Thanks.

**Review by zhztheplayer — APPROVED** [review]

File: dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "$note": "Automatically generated by generateFeatures.ps1",
   "name": "aws-sdk-cpp",
-  "version": "1.11.169",
-  "port-version": 2,
+  "version": "1.11.285",
```
Comment (marin-ma) [inline dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json:4]: Do you know the reason why the version was upgraded to 1.11.285 here but in Velox it was 1.11.321? https://github.com/facebookincubator/velox/pull/9756

Comment (marin-ma) [reply dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json:4]: Seems like they don't have 1.11.321 in ports. https://vcpkg.link/ports/aws-sdk-cpp/versions The nearest version is 1.11.321

Comment (PHILO-HE) [reply dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json:4]: @marin-ma, yes, vcpkg skips the support for 1.11.321. If there is no reason that Velox must use 1.11.321, maybe, we can upgrade it to a newer version, e.g., 1.11.474, for both Velox and Gluten vcpkg.

Comment (PHILO-HE) [reply dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json:4]: @marin-ma, it was just upgraded to the latest release (it's 1.11.321 at that time point) in Velox. But I guess we were not aware that vcpkg doesn't support this version, so use a lower version (1.11.285) in Gluten instead. So it's ok to use 1.11.285 or some other higher version at both sides. Maybe, it's better to change Gluten & velox to use a same higher version.

Comment (zhouyuan) [reply dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json:4]: https://github.com/apache/incubator-gluten/pull/9169

**Pull request merged**